spacemesh/v2alpha1/node.proto

```diff
@@ -13,10 +13,11 @@ message NodeStatusResponse {
     SYNC_STATUS_SYNCED = 3;
   }
   SyncStatus status = 2;
-  uint32 synced_layer = 3; // the last layer node has synced
-  uint32 verified_layer = 4; // the last layer node has verified
-  uint32 head_layer = 5; // head layer is the tip
-  uint32 head_block = 6;
+  uint32 synced_layer = 3; // last layer node has synced
+  uint32 verified_layer = 4; // last layer node has verified
+  uint32 head_layer = 5; // current chain head; the last layer the node has gossiped or seen
+  uint32 current_layer = 6; // current layer, based on clock time
+  uint32 head_block = 7;
```

Comment:
Blocks don't have numeric IDs. Let's rename it `head_block_id` and make it either a data blob (binary) or else a string.